pkg/process/net/common.go

```diff
@@ -162,6 +162,37 @@ func (r *RemoteSysProbeUtil) GetConnections(clientID string) (*model.Connections
 	return conns, nil
 }
 
+// GetPing returns the results of a ping to a host
+func (r *RemoteSysProbeUtil) GetPing(clientID string, host string, count int, interval time.Duration, timeout time.Duration) ([]byte, error) {
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/%s?client_id=%s&count=%d&interval=%d&timeout=%d", pingURL, host, clientID, count, interval, timeout), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Set("Accept", "application/json")
+	resp, err := r.httpClient.Do(req)
+	if err != nil {
```

Comment:
nit: the `if ... else if` construct can be flattened to only be something like 
```go
if err != nil {
  ...
}

if resp.StatusCode == http.StatusBadRequest {
  ...
}
...
```
... since each of the blocks returns.